js/anims-lasers.js

```diff
@@ -0,0 +1,30 @@
+$(function() {
+    $('.city').click(function () {
+        shootLaser($(this))
+    })
+})
+
+document.addEventListener('keypress', function(e) {
+    switch (true) {
+        case (e.key === 'q'):
+            shootLaser($('#city-1'))
+            break
+        case (e.key === 'w'):
+            shootLaser($('#city-2'))
+            break
+        case (e.key === 'e'):
+            shootLaser($('#city-3'))
+            break
+        case (e.key === 'r'):
+            shootLaser($('#city-4'))
+            break
+    }
+})
+
+function shootLaser($city) {
+    var cityid = $city.data('city')
+    $city.attr('src','img/cities-' + cityid + '-laser.png')
```

Comment:
this could be done with a class change to update the background image rather than changing the img tag, which means you can control the images from CSS and improve your separation of concerns

Reply:
fixed in 8000153